src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -1826,10 +1833,6 @@ applyTransaction (Application& app, OpenView& view,
     if (retryAssured)
         flags = flags | tapRETRY;
 
-    if ((app.getHashRouter ().getFlags (txn->getTransactionID ())
-            & SF_SIGGOOD) == SF_SIGGOOD)
-        flags = flags | tapNO_CHECK_SIGN;
```

Comment:
I'm leaving this instance out because when the Transactor calls `checkValidity`, the same flags will get checked, so this instance is redundant.